server/routes/users.js

```diff
@@ -1,11 +1,33 @@
 const express = require("express");
 const router = express.Router();
 const User = require("../models").User;
+const Project = require("../models").Project;
 
 router.get("/", function(req, res, next) {
   try {
-    User.findAll().then(users => {
-      res.send(users);
+    User.create({
+      name: "albert",
+      email: "albertkong@gmail.com"
+    }).then(user => {
+      user
+        .createProject({
+          title: "Dan's Project",
+          desc: "Wowowow",
+          photos: ["img1", "img2"]
+        })
+        .then(console.log("workded"));
+    });
+  } catch (err) {
+    console.log(err);
+  }
+});
+
+router.get("/all", function(req, res, next) {
+  try {
+    User.findAll({
+      include: [Project]
+    }).then(users => {
+      console.log(users[0].Projects);
```

Comment:
These routes are just used for testing right and will be modified in the future? Looks like you have the right relationships that we need

Reply:
Yup these routes are just for testing, and will be removed in the future. Okay great :] thanks for the review!